app/src/main/kotlin/at/bitfire/davdroid/ui/setup/LoginScreenModel.kt

```diff
@@ -80,6 +80,8 @@ class LoginScreenModel @Inject constructor(
                     ?: loginInfo.baseUri?.host
                     ?: ""
                 updateAccountNameAndEmails(initialAccountName, emails)
+                if (loginTypeUiState.loginType == NextcloudLogin)
+                    updateGroupMethod(GroupMethod.CATEGORIES)
```

Comment:
We shouldn't hardcode properties of specific login types here. (And other login methods may set a group method, too.)

I think we can

- unconditionally set the UI state: group method to `loginInfo.suggestedGroupMethod` here, and
- pass `suggestedGroupMethod = GroupMethod.CATEGORIES` where the Nextcloud's LoginInfo is created (seems that this is currently not the case).

Reply:
ah, smart! I overlooked `suggestedGroupMethod` :)